gallery/how_to/work_with_microtvm/micro_aot.py

```diff
@@ -110,11 +121,11 @@
 
 ######################################################################
 # Create a microTVM project
-# -----------------------
+# -------------------------
 #
-# Now that we have the comipled model as an IRModule, we need to create a project
-# with the compiled model in microTVM. To do this, we use Project API. We have defined
-# CRT and Zephyr microTVM template projects which are used for X86 CPU and Zephyr platforms
+# Now that we have the compiled model as an IRModule, we need to create a firmware project
+# to use the compiled model with microTVM. To do this, we use Project API. We have defined
+# CRT and Zephyr microTVM template projects which are used for x86 CPU and Zephyr platforms
```

Comment:
would "Zephyr platform (singular)" be better here? I know the boards in Zephyr by themselves can be considered also "platforms" but since we are using the Project API and in the TVMC we consider Zephyr and Arduino platforms. Or even s/platforms/boards/ ?

Reply:
boards sounds good